package.json

```diff
@@ -13,7 +13,8 @@
     "dist-all": "build -mwl --x64 --ia32",
     "pack": "build --dir",
     "dist": "build --x64 --ia32",
-    "test": "mocha test"
+    "test": "eslint . && mocha test",
```

Comment:
I don't have eslint globally. Here is an article I found that proves that npm adds node_modules/.bin/ to the PATH environment variable. And tests passed on the build agent.